.gitignore

```diff
@@ -263,4 +263,4 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
```

Comment:
When pushing commit, I chose to ignore a post file which added an unnecessary single file line here. I removed it instead of reverting back. no other changes here.